src/components/Attachments/AttachmentCarousel/Pager/index.tsx

```diff
@@ -146,14 +150,11 @@ function AttachmentCarouselPager(
 
     const carouselItems = items.map((item, index) => (
         <View
-            key={item.source}
+            key={item.source as string}
```

Comment:
Avoid type assertion. `source` is not always a string. Fallback to `reportActionID` then to `index`